nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -14,13 +14,14 @@ categoriesConfig: {
   "ServerRestApi": "request-response"
 }
 
-managementModelDir: "model"
```

Comment:
It should be able to run this configuration from docker. You want to enforce user using docker to pass all this values? Please keep in mind, that we are on of users of this config in docker - on our staging env.

Reply:
As I remeber docker uses `docker-application.conf` file..  This is only dev configuration for IDEA. I have seen thath most of people have problem with run nk from IDEA. Most of problems are envs with these paths..